pkg/controller/node/nodecontroller.go

```diff
@@ -141,6 +141,8 @@ type NodeController struct {
 	daemonSetStore listers.StoreToDaemonSetLister
 	// allocate/recycle CIDRs for node if allocateNodeCIDRs == true
 	cidrAllocator CIDRAllocator
+	// manages taints
+	taintManager *TaintController
```

Comment:
Yup - I'm applying all comments today. @timothysc - do you have a piece of email I can paste here (I wasn't taking part in those discussions)